### DISCUSSION TITLE
New notifications are in read status, although I have not open them.

### DISCUSSION THREAD

**Discussion by yunnysunny** [community/community#57504] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I open the notification button , and find some notifications I have not read before, but there are in read status. In other words, 
I can not receive the new notification message now.

**Comment by DSDmark** [comment]
Hi @yunnysunny,

You wrote:

> I can not receive the new notification message now.

But, you are seeing a notification like this:

![image](https://github.com/community/community/assets/85744816/e35a3a7b-69f4-4648-bb7c-9e84f6b38ac6)

In that case, is called **Ghost notification**. This discussion is already going on community: #6874

**Reply by yunnysunny** [reply]
I can not see the blue dot on the small bell button.
![image](https://github.com/community/community/assets/734818/57ef3326-167c-4b9c-9ac9-07e5087e0581)

**Reply by DSDmark** [reply]
@yunnysunny, What do you means by not seeing **blue dot**? If above problem which is about **Ghost notification,** if you're not facing that issue. Then, explain more about the details, and you can add some screenshot and error message which you are facing.

**Reply by yunnysunny** [reply]
![image](https://github.com/community/community/assets/734818/eb3939ba-0468-4f32-988e-bc6674c7613c)
You can see there is no sign  on the bell button to tell me that I have a new notification. And the status of current discussion is read although I have not read it

**Reply by DSDmark** [reply]
I'm not sure, why this happened. It's similar to **Ghost notification**, I guess. You could try these steps. It may solve your problem:-

1. [**steps**](https://github.com/orgs/community/discussions/6874#discussioncomment-1872442)
2. [**steps**](https://github.com/orgs/community/discussions/6874#discussioncomment-2859125)

And also change your **label** into **Bug** and **Notifications** of this discussion.

**Comment by roncli** [comment]
I too am having this problem.  This is not ghost notifications.  Often it happens from dependabot notifications.  Example, the top 3 notifications I've never touched:

![image](https://github.com/community/community/assets/498929/355675e6-289f-4829-bd92-ed63fe751839)
